internal/flag/flags.go

```diff
@@ -24,6 +24,12 @@ const (
 	Secret                          = "secret"                          // Secret flag
 	ProjectID                       = "projectId"                       // ProjectID flag to use a project ID
 	AuthDB                          = "authDB"                          // AuthDB flag
+	Hostname                        = "hostname"                        // Hostname flag
+	Port                            = "port"                            // Port flag
```

Comment:
wow we did not have this yet?

Reply:
yes, that's really strange indeed. I was sure they were already there 🤷‍♂️